docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md

```diff
@@ -850,6 +850,68 @@ const [sum] = controller.parseQueryResponse(response);
 console.log(sum);
 ```
 
+## Explicit decoding / encoding of values
+
+When needed, you can use the [`BinaryCodec`](https://multiversx.github.io/mx-sdk-js-core/v13/classes/BinaryCodec.html) to [decode and encode values](/developers/data/serialization-overview/) manually.
+
+### Decoding a custom type
+
+Example of decoding a custom type (a structure) called `DepositEvent` from binary data:
+
+```
+import { BinaryCodec } from "@multiversx/sdk-core";
+
+const depositCustomType = abi.getCustomType("DepositEvent");
```

Comment:
maybe also instantiate the `abi` object here?

Reply:
Fixed.